lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +129,9 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        for index, token in enumerate(tokens):
```

Comment:
read about the `start` argument of the `enumerate` function

Reply:
done